docs/DeveloperGuide.md

```diff
@@ -148,52 +149,22 @@ The `Exceptions` component currently consists of eleven (11) custom Exceptions,
-* **`InvalidDateException`**: Is thrown when user input, where a date is expected, does not
-  match the `dd-mm-yy` format.
-  * Error Message: `"Sorry my friend, the date must be in the form 'dd-mm-yy'."`
-* **`InvalidInputException`**: Is thrown when an unrecognised command is input into FridgeFriend,
-  or when the List Command does not recognise the secondary input string.
-  * Error Message: `"Sorry my friend, please give a valid input."`
-* **`InvalidQuantityException`**: Is thrown when user input is not an integer where an integer is expected for
-  `Quantity` related functions, or when excess quantity of food is removed from the Fridge, where the quantity
-  of food in the fridge remaining would be negative.
-  * Error Message: `"Sorry my friend, the quantity QUANTITY_INPUT must be a number."`
-  * Error Message: `"Not enough in fridge to remove!"`
-
-_Exceptions related to Food:_
-
-* **`FoodNameNotFoundException`**: Is thrown when the input string contains the name of a Food that is
-  not found in the Fridge, where the name of a food existing in the Fridge is expected.
-  * Error Message: `"Food specified not found."`
-* **`InvalidFoodCategoryException`**: Is thrown when user input is not a valid `FoodCategory`,
-  where a valid `FoodCategory` is expected. Users can check `help` or the
-  [User Guide](https://ay2021s2-cs2113-t10-1.github.io/tp/UserGuide.html).
-  * Error Message: `"Sorry my friend, FOOD_CATEGORY is not a valid category."`
-* **`InvalidFoodLocationException`**: Is thrown when user input is not a valid `FoodLocation`,
-  where a valid `FoodLocation` is expected. Users can check `help` or the
-  [User Guide](https://ay2021s2-cs2113-t10-1.github.io/tp/UserGuide.html).
-    * Error Message: `"Sorry my friend, FOOD_LOCATION is not a valid category."`    
-* **`RepetitiveFoodIdentifierException`**: Is thrown when user attempts to add duplicate Food to the Fridge in a
-  different location or with a different expiry date. This is not allowed as of v2.0.
-  * Error Message: `"Sorry my friend, you have added this food before but in a different location or have different expiry dates.
-      Please specify another foodname."`
-
-_Exceptions related to file storage:_
-
-* **`StorageLoadingException`**: Is thrown when an error occurred during loading of the saved data.
-  * Error Message: `"There was an error loading the data for FridgeFriend!"`
-* **`StorageSavingException`**: Is thrown when an error occurred during saving of the current data.
-  * Error Message: `"There was an error saving the data for FridgeFriend!"`
+* Invalid User Input
+  * Thrown when a raw user input cannot be parsed into a valid value
+  * Includes: `EmptyDescriptionException`, `InvalidDateException`, `InvalidInputException`, `InvalidQuantityException`
+* Food Errors
+  * Thrown when a parameter for a food item is invalid
+  * Includes: `FoodNameNotFoundException`, `InvalidFoodCategoryException`, `RepetitiveFoodIdentifierException`
```

Comment:
We are missing InvalidFoodStorageLocation Exception and the newest Minimum Quantity exception

Reply:
minquantity will be done by @SimJJ96 
edited for for location